README.org

```diff
@@ -69,6 +65,40 @@ There is an option to have your user credentials (email address and password) sa
 The default is not to do this because if not properly configured it would save these unencrypted which is not a good default to have.
 Customize the variable =mastodon-auth-source-file= if you want to enable this feature.
 
+*** 2 Factor Auth
+**** Overview
+After setting up =mastodon-instance-url= set the following in your
+=.emacs= or =customize=:
+
+1. =mastodon-auth-mechanism=
+2. =mastodon-oauth2-client-id=
+3. =mastodon-oauth2-client-secret=
+
+**** Do the following to obtain OAuth2 credentials:
```

Comment:
Isn't this exactly what `mastodon-client--register` does?  Why does the user have to do that manually now?

Reply:
I see, missed that one. I shall refactor.